docs/en/qgc-user-guide/setup_view/joystick.md

```diff
@@ -45,6 +45,10 @@ To configure a joystick:
 
    The joystick is _enabled_ as the last step of the calibration process.
 
+    ::: warning
+    On some radiocontrollers, the calibration process does not work because of wrong channels mapping. You can follow [supported joysticks section](#supported-joysticks) for more information.
```

Comment:
Can you change the wording here from:
`On some radiocontrollers, the calibration process does not work because of wrong channels mapping.`
to:
`On some controllers the calibration process does not work because of incorrect channel mappings.`